src/style.css

```diff
@@ -1 +1,80 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  /* stylelint-disable-next-line string-quotes */
```

Comment:
try to not use stylelint-disable

Reply:
@Viktor-Kost if not used, throws an error in css

Reply:
so? you should fix an error instead of using stylelint-disable, just remove string quotes